app/assets/stylesheets/base/_utilities.scss

```diff
@@ -1,3 +1,7 @@
+.hidden {
+  display: none !important;
```

Comment:
Unfortunately it is needed. Some of my functionality overwrites the display: none without the `!important`